runner/scripts/3-completeAssignments.ts

```diff
@@ -1,29 +1,28 @@
+import * as Path from 'path';
 import Puppeteer from 'puppeteer';
```

Comment:
Why only puppeteer?

Reply:
What others would you recommend? IMO, the primary purpose of running the full DA cycle is to test all the collect and analyze code/plugins. It doesn't seem like anything other than Puppeteer is needed.

Reply:
I just wasn't sure why it's puppeteer instead of secret agent

Reply:
Ah, interesting. I guess I think of Puppeteer as a bit closer to the bare metal. So I was thinking it would reduce the chances of pulling SA bugs into the mix when trying to test new collect/analyze code.

Perhaps I should have gone with a pure-devtools approach (closer to bare metal) or SA.

Thoughts?

Reply:
I think pure devtools makes more sense than Puppeteer. But I also think it makes a lot of sense to use SA here to have a bit of our own full-circle going on...

Reply:
Pure devtools is a bit of a pain to get clicks and navigation properly tracking though... so probably would go with SecretAgent here, or just keep your choice of puppeteer purely from a "least likely to have bugs" perspective.

Reply:
Cool to do in another PR?

Reply:
sure